### ISSUE TITLE
Twenty Sixteen: Comment Form social icons don't have contrast with dark backgrounds

### ISSUE THREAD

**Issue by liviopv** [Automattic/themes#3062]
## Steps to replicate

1. Set a dark background on a site with Twenty Sixteen

## Result

Because most of the icons (WordPress, Twitter, Facebook) are displayed in a dark shade of blue with a transparent background, the icons don't have a lot of contrast with the background.

![Screenshot](https://d.pr/i/O0r1PH+)
Screenshot: https://d.pr/i/O0r1PH

## Expected

We should display the icons with its own background to create contrast, like we do in other themes

This if from Barnsbury:
![Screenshot](https://d.pr/i/q44Snc+)
Screenshot: https://d.pr/i/q44Snc

* User report: #3510459-zd

**Comment by danieldudzic** [comment]
After some investigating, this seems to be a 'rabbit-hole' of an issue.

The entire comment form styling should be re-worked to handle gracefully changing color palettes.

At the moment we are running into contrast issues:
<img width="1042" alt="comment" src="https://user-images.githubusercontent.com/905781/105432283-a4b27100-5c57-11eb-9771-a6ceed1fb67f.png">

We could also just remove the custom styling and rely on the default styles.

**Comment by mrfoxtalbot** [comment]
I can still reproduce this. While not great, the contrast is acceptable when the default palettes are used:
![Screen Shot on 2022-06-23 at 18:09:52](https://user-images.githubusercontent.com/4452464/175345566-b4a12e15-1365-45e8-8cd8-6b32974f6af8.png)
![Screen Shot on 2022-06-23 at 18:09:37](https://user-images.githubusercontent.com/4452464/175345575-e55c81f3-94fd-4212-9bb4-00778a2606e0.png)
![Screen Shot on 2022-06-23 at 18:09:24](https://user-images.githubusercontent.com/4452464/175345577-9b9652ee-e911-402e-aec6-71b558e0f75b.png)
![Screen Shot on 2022-06-23 at 18:09:13](https://user-images.githubusercontent.com/4452464/175345582-474d2787-2ab8-4b8e-92c3-32634bc39d0b.png)
![Screen Shot on 2022-06-23 at 18:08:58](https://user-images.githubusercontent.com/4452464/175345585-5d208a86-3cea-4005-a68f-924a98d490fa.png)

I am going to mark this as cannot reproduce but please feel free to reopen if needed @liviopv
